tests/core/test_policies.py

```diff
@@ -337,6 +337,27 @@ def create_policy(self, featurizer, priority):
     def test_similarity_type(self, trained_policy):
         assert trained_policy.similarity_type == "inner"
 
+    def test_ranking_length(self, trained_policy):
+        assert trained_policy.ranking_length == 10
+
+    def test_normalization(self, trained_policy, tracker, default_domain, monkeypatch):
```

Comment:
I wonder why tests past, if you didn't assign output of normalize function to confidence/message_sim

Reply:
actually, I don't see a test for checking that norm is 1